pymatgen/electronic_structure/cohp.py

```diff
@@ -343,12 +343,48 @@ def get_cohp_by_label(self, label):
         else:
             try:
                 return Cohp(efermi=self.efermi, energies=self.energies,
-                            cohp=self.all_cohps[label].get_cohp(spin=None, integrated=False), are_coops=self.are_coops,
+                            cohp=self.all_cohps[label].get_cohp(spin=None, integrated=False),
+                            are_coops=self.are_coops,
                             icohp=self.all_cohps[label].get_icohp(spin=None))
 
             except KeyError:
                 print("The label does not exist")
 
+    def get_summed_cohp_by_labellist(self, labellist, divisor=1):
```

Comment:
I would suggest putting an underscore in `label_list`, both in method name and arg name.